project-example/src/connection_handler.rs

```diff
@@ -93,9 +94,10 @@ pub fn get_connection(opt: Opt, components: ComponentDatabase) -> Result<WorkerC
 
             WorkerConnection::connect_alpha_locator_async(
                 &alpha_locator,
-                &ConnectionParameters::new(worker_type, components)
+                &ConnectionParameters::new(worker_type)
                     .using_tcp()
-                    .using_external_ip(true),
+                    .using_external_ip(true)
+                    .generate_component_vtable(),
```

Comment:
Nit: Would prefer naming this to something that isn't "vtable", as that's an implementation detail. Maybe `enable_internal_serialization()`

Reply:
Yup - seems sensible, will change